examples/hello-world/ml-to-fl/tf/README.md

```diff
@@ -75,16 +73,14 @@ python3 ./code/cifar10_tf_multi_gpu_original.py
 To transform the existing code to FL training code, we made the following changes:
 
 1. Import NVFlare Client API: ```import nvflare.client as flare```
-2. Import how to load/dump flatten weights: ```from nvflare.app_opt.tf.utils import get_flat_weights, load_flat_weights```
-3. Initialize NVFlare Client API: ```flare.init()```
-4. Receive aggregated/global FLModel from NVFlare side: ```input_model = flare.receive()```
-5. Load the received aggregated/global model weights into the model structure: ```load_flat_weights(net, input_model.params)```
-6. Evaluate on received aggregated/global model to get the metrics for model selection
-7. Construct the FLModel to be returned to the NVFlare side: ```output_model = flare.FLModel(params=get_flat_weights(net), xxx)```
-8. Send the model back to NVFlare: ```flare.send(output_model)```
-
-Notice that we need to flatten/unflatten the model weights because NVFlare server-side aggregators now
-only accept a ``dict`` of arrays.
+2. Initialize NVFlare Client API: ```flare.init()```
+3. Receive aggregated/global FLModel from NVFlare side: ```input_model = flare.receive()```
+4. Load the received aggregated/global model weights into the model structure: ```model.get_layer(k).set_weights(v)```
+5. Evaluate on received aggregated/global model to get the metrics for model selection
+6. Construct the FLModel to be returned to the NVFlare side: ```output_model = flare.FLModel(params={layer.name: layer.get_weights() for layer in model.layers}, xxx)```
+7. Send the model back to NVFlare: ```flare.send(output_model)```
+
+Notice that we need to get / load the model weights as a ``dict`` of arrays because we want to reuse existing NVFlare components.
```

Comment:
seems the same process as above? shall we just say "same changes for multi-gpu"?